src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+@import "./utils/variables";
+@import "../utils/functions";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $margin;
+  width: widthContainer($cell-size);
+
+  @for $day from 28 through 31 {
+    &--length-#{$day}>:nth-child(n + #{1+$day}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    transition-duration: 500ms;
+    transition-property: transform;
```

you can just use transition instead two property
also, create variable for transition Duration value
```suggestion
    transition: transform $transitionDuration;
```